src/tech/yk_structure.md

```diff
@@ -47,3 +47,6 @@ There are a few implementation details to note:
 - Due to the separate compilation of the workspaces, some code will be
   duplicated. To avoid collisions of unmangled symbols, the internal workspace
   is compiled into a shared object.
+
+- Consumers of the JIT libraries must use the abort strategy. No attempt is
```

Comment:
The second sentence doesn't really make sense in relation to the first -- can we rephrase or expand it a bit?

Reply:
The crux of it is, if you use the abort strategy, you will never see unwinding.

I'll try a rephrase.

Reply:
Does b0ebb25 help?

Reply:
Yep!